### PR TITLE
Cleaning some @param in order to make Doxygen work

### PR DESCRIPTION
**Pull request by loumalouomega**
This PR mainly removes the ":" from the `@param` in the core. Still much work to do in order to make a good and coherent Doxygen compilation. The current version (core only) can be seen here:

http://loumalouomega.github.io

The fancy (http://mcss.mosra.cz/doxygen/) version will not compile until we clean a little bit more the core

### CODE REVIEW

File: kratos/modified_shape_functions/modified_shape_functions.h [hunk 1]
```diff
@@ -296,11 +296,11 @@ class KRATOS_API(KRATOS_CORE) ModifiedShapeFunctions
     * @return rInterfaceShapeFunctionValues: Matrix containing the computed shape function values.
     * @return rInterfaceShapeFunctionsGradientsValues: std::vector containing the shape functions gradients values.
     * @return rInterfaceWeightsValues: Vector containing the Gauss pts. weights (already multiplied by the Jacobian).
-    * @param rInterfacesVector: std::vector of intersection point based geometries where the values are to be computed.
-    * @param rParentGeometriesVector: std::vector of subdivisions point based parent geometries.
-    * @param rInterfacesParentIdsVector: std::vector containing the parent ids of each interface geometry.
-    * @param rPmatrix: reference to the interface interpolation matrix
-    * @param IntegrationMethod: Desired integration quadrature.
+    * @param rInterfacesVector stdvector of intersection point based geometries where the values are to be computed.
```
Comment (pooyan-dadvand) [inline kratos/modified_shape_functions/modified_shape_functions.h]: Why you are taking out the std::vector double points? Is there anyway to keep it?

File: kratos/utilities/binbased_nodes_in_element_locator.h [hunk 2]
```diff
@@ -121,11 +121,19 @@ class BinBasedNodesInElementLocator
...
+    * @param work_results The point iterator with the points to be studied
+    * @param work_distances The iterator containing the relative distances
+    * @param work_point The node of study
+    * @return counter: The number of iterations needed
     */
     unsigned int FindNodesInElement(Element::Pointer& pelement,
                                     boost::numeric::ublas::vector<int>& positions,
                                     Matrix& Nmat,
-                                    const unsigned int& max_results,
+                                    const unsigned int max_results,
```
Comment (pooyan-dadvand) [inline kratos/utilities/binbased_nodes_in_element_locator.h:136]: You may not change an argument type of a class in a PR for doxygen

**Review by pooyan-dadvand — CHANGES_REQUESTED** [review]
I would suggest you to definitely avoid code changes in a documentation PR. It increments considerably the revision times and the possibility of adding new bugs due to poor reviews

File: kratos/modified_shape_functions/modified_shape_functions.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/modified_shape_functions/modified_shape_functions.h]: I used a script to speed up the work, I checked out, but maybe I made some mistake

File: kratos/utilities/binbased_nodes_in_element_locator.h (hunk 2 above)
Comment (loumalouomega) [reply kratos/utilities/binbased_nodes_in_element_locator.h:136]: Sorry, not again

File: kratos/utilities/kratos_log.cpp [hunk 3]
```diff
@@ -47,11 +47,13 @@ namespace Kratos
         return (*mpInstance);
     }
     
+    // Default constructor
```
Comment (RiccardoRossi) [inline kratos/utilities/kratos_log.cpp:50]: @pooyan-dadvand are u working on this file?

**Review by RiccardoRossi — COMMENTED** [review]
i have no remarks, but Pooyan's point are valid.

i let him approve this

File: kratos/utilities/kratos_log.cpp (hunk 3 above)
Comment (loumalouomega) [reply kratos/utilities/kratos_log.cpp:50]: I think that the current in development logger is in a different folder `io`

Comment (pooyan-dadvand) [reply kratos/utilities/kratos_log.cpp:50]: No, this is a deprecated version to be removed. (we are transferring macros from this one)

**Review by pooyan-dadvand — APPROVED** [review]

**Comment by loumalouomega** [comment]
Ok, thanks

**Pull request merged**